packages/shared-components/UpdateMediaContent/index.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Lightbox from 'react-images';
+import { IdTag, Image, MultipleImages } from '@bufferapp/components';
+import { grayLighter, gray } from '@bufferapp/ui/style/colors';
+import { borderRadius } from '@bufferapp/ui/style/borders';
+
+const MediaWrapper = styled.div`
+  position: relative;
+  cursor: pointer;
+`;
+
+const ImageWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 9rem;
+  height: 9rem;
+  border-radius: ${borderRadius};
+  border: 1px solid ${gray};
+  background: ${grayLighter};
+`;
+
+const ThumbnailTag = styled.span`
+  position: absolute;
+  bottom: 0.7rem;
+  left: 0.7rem;
+`;
+
+const UpdateMediaContent = ({
+  onImageClick,
+  imageSrc,
+  imageUrls,
+  isLightboxOpen,
+  onImageClickPrev,
+  onImageClickNext,
+  onImageClose,
+  currentImage,
+  tag,
+  type,
+}) => {
+  const isMultipleImage = type === 'multipleImage';
+  const isSingleImage = type === 'image' || type === 'video';
+  const images = isMultipleImage
+    ? imageUrls?.map(url => ({ src: `${url}` }))
+    : [{ src: `${imageSrc}` }];
```

Comment:
could this be made slightly cleaner up by doing the following, then you only have the image processing happening in one place consistently?
```suggestion
  const imageArray = isMultipleImage ? imageUrls : [imageSrc];
  const images = imageArray?.map(url => ({ src: `${url}` }));
```